src/libvirtApi/domain.js

```diff
@@ -1054,3 +1056,16 @@ export async function domainReplaceSpice({ connectionName, id: objPath }) {
         throw ex; // not-covered: see above
     }
 }
+
+export async function domainAddTPM({ connectionName, vmName }) {
+    const args = ["virt-xml", "-c", `qemu:///${connectionName}`, "--add-device", "--tpm", "default", vmName];
+    return cockpit.spawn(args, { err: "message", superuser: connectionName === "system" ? "try" : null })
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1448-20240219-135731-f4f73927-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)